trestle/core/commands/author/ssp.py

```diff
@@ -554,7 +556,16 @@ def _run(self, args: argparse.Namespace) -> int:
             if args.regenerate:
                 ssp, _, _ = ModelUtils.regenerate_uuids(ssp)
             ModelUtils.update_last_modified(ssp)
-
+            # validates if ssp doesn´t have a profile a provides warning
+            if not ssp.import_profile.href:
+                logger.warning('No profile given for current SSP. Model is invalid')
```

Comment:
Please use `logger.error` for this and next warning. Since program quits after this log it should be an error

Reply:
Done